packages/angular/src/generators/ngrx/files/classes-syntax/__directory__/__fileName__.reducer.spec.ts__tmpl__

```diff
@@ -25,7 +23,7 @@ describe('<%= className %> Reducer', () => {
 
   describe('unknown action', () => {
     it('should return the previous state', () => {
-      const action = {} as Action;
+      const action = {} as any;
```

Comment:
Why the cast to `any` here?

Reply:
With `Action`, I was getting a TS error saying that `Action` is not assignable to the exported union type of the actions because `Action` doesn't have the `payload` property and some actions of the union type do have it.

We had this cast to `any` before, it was changed to `Action` recently to solve the EsLint warning, but it did introduce this TS error.

Reply:
👍🏿